### ISSUE TITLE
Keras API - Initial design - 2 weeks timeline

### ISSUE THREAD

**Issue by yanndupis** [tf-encrypted/tf-encrypted#479]
Child of #470 

As a first step we want to tackle the following features: 
- [ ] tfe.keras.layers:
    - [x] base Layer class:
    - [x]  tf.keras.layers.Dense
    - [x] tf.keras.layers.Conv2D
    - [x] tf.keras.layers.MaxPooling2D
    - [x] tf.keras.layers.AvgPooling2D
    - [ ] tf.keras.layers.BatchNormalization
    - [ ]  tf.keras.layers.Reshape
    - [x] tf.keras.layers.Flatten
    - [x] tf.keras.layers.InputLayer
    - [x] tf.keras.layers.Activation
        - [x] ReLU
        - [x] Tanh 
        - [x] Sigmoid
- [x] Very basic initializer to initialize kernels
- [ ] tfe.keras.models.Sequential
    - [x] Sequential class
    - [ ] predict method 
    - [ ] load_weights method 

By completing this issue, we should be able to support private predictions for convolution models such as VGG16.

**Comment by mortendahl** [comment]
@yanndupis want to move the remaining items into new issues (say, an issue for each missing layer, and one for adding predict+loading to sequential)? then we can close this guy up

**Comment by mortendahl** [comment]
Closing this issue, we have what we need for the PySyft integration.
